Gordon360/Services/ScheduleService.cs

```diff
@@ -89,5 +89,74 @@ public async Task<IEnumerable<ScheduleViewModel>> GetScheduleFacultyAsync(string
                 END_TIME = x.END_TIME
             });
         }
+
+
+        /// <summary>
+        /// Fetch the schedule item whose id and session code is specified by the parameter
+        /// </summary>
+        /// <param name="username">The AD Username of the instructor</param>
+        /// <param name="sessionID">The session of the instructor</param>
+        /// <returns>StudentScheduleViewModel if found, empty array if not found</returns>
+        public async Task<IEnumerable<ScheduleViewModel>> GetSpecificScheduleFacultyAsync(string username, string sessionID)
```

Comment:
Your faculty and student services should be able to be done in a single function based on `ACCOUNT_TYPE`